src/components/Label/Label.stories.js

```diff
@@ -1,53 +1,49 @@
 /* @flow */
 import React from 'react';
+import R from 'ramda';
+
 import { storiesOf, action, linkTo } from '@kadira/storybook';
 import {
-   withKnobs,
-   text,
-   boolean,
-   number,
-   select,
+  withKnobs,
+  text,
+  boolean,
+  number,
+  select,
 } from '@kadira/storybook-addon-knobs';
 
+import { PositionEnum, SizeEnum } from '../../assets/constants';
+const positionOptions = R.invertObj(PositionEnum);
+const sizeOptions = R.invertObj(SizeEnum);
+
 import Label from '.';
 
 const stories = storiesOf('Label', module);
 stories.addDecorator(withKnobs);
 stories.addDecorator(story => (
```

Comment:
let's render the components in the top left corner. It's better to spot the actual padding and margin that is applies. so i suggest to drop this decorator to be consistent.